content/en/security/cloud_security_management/setup/_index.md

```diff
@@ -69,28 +69,74 @@ CSM is available in three packages: [CSM Enterprise][1], [CSM Pro][2], and [CSM
 **Notes**: 
 
 - You can enable features that aren't included in your package at any time by following the instructions on the [CSM Setup page][4].
```

Comment:
```suggestion
**Note**: You can enable features that aren't included in your package at any time by following the instructions on the [CSM Setup page][4].
```

Reply:
I think we should consider moving this note above the table, especially now that it only contains the one sentence.

Reply:
Oh it's actually still 2 sentences so I'll leave it for now (not sure why it cut it off here)

```
**Notes**: 
You can enable features that aren't included in your package at any time by following the instructions on the [CSM Setup page][4].
For setup instructions, see [Cloud Security Management Identity Risks][5] and [Setting up Cloud Security Management Vulnerabilities][6].
```

Reply:
Got it, noted.